core/build.gradle.kts

```diff
@@ -1,3 +1,5 @@
+import org.jetbrains.kotlin.js.inline.clean.removeUnusedImports
+
```

Comment:
Do we need this import here? I don't see it being used in the file.

Reply:
Good catch. Thank you!

Updated.